includes/options/class-amp-options-menu.php

```diff
@@ -670,6 +681,81 @@ public function render_stories_export() {
 		<?php
 	}
 
+	/**
+	 * Render the stories settings section.
+	 *
+	 * @since 1.3
+	 */
+	public function render_stories_settings() {
+		$this->render_stories_settings_page_advance();
```

Comment:
When the stories experienced is **disabled** on the AMP settings screen, the "Stories Settings" section should be **hidden**. This is currently the case for the "Stories Export" section.

There is some inline JS on the settings page that handles this.

Screenshots:

![Screenshot 2019-09-11 at 19 22 45](https://user-images.githubusercontent.com/841956/64720070-de9f2f00-d4c9-11e9-8996-d13e0b96a2d3.png)
![Screenshot 2019-09-11 at 19 22 38](https://user-images.githubusercontent.com/841956/64720071-df37c580-d4c9-11e9-8d63-d8e1e5c71e93.png)

Speaking of which, it makes sense to incorporate the "Stories Export" setting into this "Stories Settings" section so both settings (export  and page advance) are under the "Stories Settings" section.

Mockup:

![Screenshot 2019-09-11 at 19 27 05](https://user-images.githubusercontent.com/841956/64720216-32117d00-d4ca-11e9-99d5-4f83ce4614e2.png)

Reply:
@swissspidy Incorporating the export setting into Stories Settings may be confusing in tandem with the additional request you made:

> Here, I'd expect a similar help text: "These settings are applied to new stories only."

The logical place for this text is on top of all the settings, but it's not true the export setting is applied to new stories only. That's a universal setting. It's only these newly added settings that only apply to new stories.

I'd suggest we keep them separate - at least for the time being. And maybe explore this concept of "defaults" vs. "settings" more from the UX perspective later on.